addons/website_slides/models/slide_question.py

```diff
@@ -9,6 +9,7 @@ class SlideQuestion(models.Model):
     _name = "slide.question"
     _rec_name = "question"
     _description = "Slide Quiz Question"
+    _order = "sequence"
 
     sequence = fields.Integer("Sequence", default=10)
```

Comment:
I would set the default at 0 to make the already existing questions coming from data or demo data to have 0 as sequence if not set. So that when you create a new question, it will be set @ sequence = 1 and won't be placed before existing questions
Same logic for answers.